CMakeLists.txt

```diff
@@ -396,7 +401,7 @@ if(KTX_FEATURE_VULKAN)
     )
     target_include_directories(
         ktx
-    PUBLIC
+    PRIVATE
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/dfdutils>
         $<INSTALL_INTERFACE:lib/dfdutils>
```

Comment:
This is also going to be needed by the app when linking with a static library which is a problem as there is currently no separate release of the dfdutils library.

Reply:
Do users of KTX need to use dfdutils (dfd.h) directly commonly?

A clean solution could be to create a CMake config for dfdutils as well and link to its CMake target.
Users who need dfdutils directly could direclty refer to it.

Reply:
Quite possibly, if the app wishes to examine the DFD. At present only a small part of the information it contains is made available through the libktx API.

I agree we should make a CMake config for it, as you suggest @atteneder. dfdutils is included via `git subrepo` from https://github.com/KhronosGroup/dfdutils. I need to push some recent additions and fixes back there and we should add the CMake config there.